crates/bindings-csharp/Codegen/Codegen.csproj

```diff
@@ -23,38 +25,24 @@
     <PackageIcon>logo.png</PackageIcon>
     <PackageReadmeFile>README.md</PackageReadmeFile>
     <RepositoryType>git</RepositoryType>
-    <RepositoryUrl>https://github.com/clockworklabs/SpacetimeDBLibCSharp</RepositoryUrl>
+    <RepositoryUrl>https://github.com/clockworklabs/SpacetimeDB</RepositoryUrl>
     <PackageLicenseFile>LICENSE</PackageLicenseFile>
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="../BSATN.Codegen/BSATN.Codegen.csproj" PrivateAssets="all" />
+    <!-- This analyzer uses some shared helpers from the BSATN.Codegen one, so include it as a regular dependency -->
+    <!-- (*not* with OutputItemType=analyzer because we don't want to use that source generator on source of Codegen itself) -->
+    <ProjectReference Include="../BSATN.Codegen/BSATN.Codegen.csproj" />
```

Comment:
sanity checking my understanding - we removed `PrivateAssets="all"` because we (now) _do_ want the contents of `BSATN.Codegen.csproj` to be available to people who include `Codegen.csproj`, is that right?

Reply:
It's just another one of "forth and back" changest that don't matter in the end.

Here it just doesn't matter one way or another, because `PrivateAssets="all"` only matters for NuGet packaging and neither Codegen nor BSATN.Codegen are packable after this PR, so having or removing that attribute has zero difference.

I'd probably keep it removed just to avoid someone asking why we need `PrivateAssets="all"` on a seemingly no-op package in the future.

Reply:
I removed `PrivateAssets="all"` on other codegen references as well for consistency & to reduce noise.